native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java

```diff
@@ -1178,4 +1178,20 @@ public static String getBTypeName(Type ballerinaType) {
         }
         return ballerinaType.getName();
     }
+
+    public static boolean isKnownRecordType(Type ballerinaType) {
+        String[] knownRecordTypes = {
+                Constants.SqlTypes.CIVIL,
+                Constants.SqlTypes.DATE_RECORD,
+                Constants.SqlTypes.TIME_RECORD
+        };
+        String typeName = getBTypeName(ballerinaType);
+        for (String type: knownRecordTypes) {
```

Comment:
We can use contains method on a list. Much cleaner

Reply:
Fixed in 2bf215b